testing/mr_disjoint_pool.cu

```diff
@@ -48,7 +49,10 @@ struct pointer_traits<alloc_id>
         return reinterpret_cast<void *>(id.alignment);
     }
 };
-}}
+
+} // end namespace detail
+
```

Comment:
Existing code is all over the place with these sorts of style issues, so there's not a firm rule.

I generally prefer a single blank line. This is what our proposed `.clang-format` file will do, and we'll fix this when we get around to setting that up since we can automate it.